core/textured-cube/main.zig

```diff
@@ -200,6 +203,12 @@ pub fn update(app: *App) !bool {
                     .one => app.core.setVSync(.none),
                     .two => app.core.setVSync(.double),
                     .three => app.core.setVSync(.triple),
+                    .left => entity_position = entity_position + zm.f32x4(-speed, 0, 0, 0),
+                    .right => entity_position = entity_position + zm.f32x4(speed, 0, 0, 0),
+                    .up => entity_position = entity_position + zm.f32x4(0, speed, 0, 0),
+                    .down => entity_position = entity_position + zm.f32x4(0, -speed, 0, 0),
+                    .z => entity_position = entity_position + zm.f32x4(0, 0, speed, 0),
+                    .x => entity_position = entity_position + zm.f32x4(0, 0, -speed, 0),
```

Comment:
totally not important but why not just `entity_position += zm.f32x4(-speed, 0, 0, 0)`

Reply:
I was probably overthinking this and going "Oh, we probably want to mimic x/y/z down the road" but it ultimately doesn't mean anything for this purpose/PR.